components/join-us-modal/step.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class Step extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {}
```

Comment:
these empty functions should not be necessary

Reply:
yup forgot to remove those 😂